jobs/build/pre-release/Jenkinsfile

```diff
@@ -138,6 +138,20 @@ node {
                 }
             }
 
+            stage("sign") {
+                if ( params.MIRROR ) {
+                    release.signArtifacts(
+			name: foo,
```

Comment:
:rofl:

Reply:
whooops

Reply:
in the `release` job this is `name: name,` and I don't actually see how `name` is getting set (but it clearly is). spooky!

otherwise this LGTM